.github/workflows/tests.yaml

```diff
@@ -62,7 +62,14 @@ jobs:
       run: tuist install --path SampleApp
 
     - name: Build & Test
-      run: tuist test --path SampleApp --os ${{ matrix.sdk }} UnitTests
+      run: tuist test --path SampleApp --os ${{ matrix.sdk }} UnitTests -- -resultBundlePath TestResult
+    
+    - name: Archive TestResult
+      uses: actions/upload-artifact@v4
+      if: ${{ failure() }}
```

Comment:
Can this be bundled into the existing `actions/upload-artifact@v4` below? Or at least move them next to each other?